applications/API/apps.py

```
@@ -7,4 +7,48 @@ class ApiConfig(AppConfig):
+            static_prefixes = (
+                "Insult",
+                "InsultCategory",
+                "Insult_view",
+                "Insult_categories",
+            )
+
+            all_prefixes = tuple(set(registry_prefixes + static_prefixes))
+            _pre_register_invalidation_labels(all_prefixes)
+        except Exception:
+            # Never block startup — metrics pre-registration is best-effort.
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)